features/support/ui/your_contributions.rb

```diff
@@ -8,6 +8,11 @@ class YourContributions < SitePrism::Page
     set_url '{/locale}/tools/wpcc/your_contributions'
     set_url_matcher(/wpcc\/\d+\/your_contributions/)
 
+    section :employee_percent,   EmployeePercentSection, '.contributions__source--employee'
+    section :employer_percent,   EmployerPercentSection, '.contributions__source--employer'
+    
     element :details, '.details__row'
+    element :contributions_description, '.t-contributions_description'
```

Comment:
I would prefer to use .t- so that it's clear this is not a class associated with any styling.